File: Resource/51900286-51900712-51900718-51900699/source demo/Web Application/ComNha_API/ComNha_API/Models/Nguyenvatlieu.cs
using System;
using System.Collections.Generic;

namespace ComNha_API.Models
{
    public partial class Nguyenvatlieu
    {
        public string MaNguyenVatLieu { get; set; } = null!;
        public string TenNguyenVatLieu { get; set; } = null!;
        public double DonGia { get; set; }
        public string DonVi { get; set; } = null!;
        public string TinhTrang { get; set; } = null!;
        public int SoLuongTonKho { get; set; }
    }
}

File: References/restaurent-management-comnha/src/main/java/tdtu/thinh6/restaurent_management_comnha/domain/Nguyenvatlieu.java
package tdtu.thinh6.restaurent_management_comnha.domain;

import java.util.Set;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.OneToMany;
import lombok.Getter;
import lombok.Setter;


@Entity
@Getter
@Setter
public class Nguyenvatlieu {

    @Id
    @Column(nullable = false, updatable = false, length = 10)
    private String maNguyenVatLieu;

    @Column(nullable = false, length = 50)
    private String tenNguyenVatLieu;

    @Column(nullable = false)
    private Double donGia;

    @Column(nullable = false, length = 50)
    private String donVi;

    @Column(nullable = false, length = 50)
    private String tinhTrang;

    @Column(nullable = false)
    private Integer soLuongTonKho;

    @OneToMany(mappedBy = "maNguyenVatLieu")
    private Set<Chitietphieunhap> maNguyenVatLieuChitietphieunhaps;

}
